onboarding_guide_outline.md

```diff
@@ -67,15 +67,31 @@ Link to snapshot of the server running
 
 Before we try to make this faster, let's measure it.
 
-Add a local_resource that records the current time, and kicks off a build.
+Tilt can run commands locally.
 
-Add code to the app that records start up time.
+We use local_resource, which lets you locally run scripts, shell code, or servers.
+
+We want to measure the time from code change to new process.
+
+First, we add a local_resource that records the current time to a source file.
+
+Second, we'll add code to the app that reads that start time and calculates how
+much time elapsed before the server came up. Our app code logs this elapsed time
+and displays it in the served HTML.
 
 Let's see it!
 
+Screenshot
+
 Current score of our naive hello world app.
 
-### Attempt #1
+You may notice that the time displayed in the Tilt sidebar is different than the
+time our app logged. That's OK! In multi-service development, there are many
+"start times" we care about -- time to build, time to schedule, time until
```

Comment:
nice, I like this! my one quibble is that it's less "start time" and more "duration" or "benchmark"?

Reply:
ya, i like benchmark better too